### PR TITLE
chapters added to adventure map

### PR DESCRIPTION
**Pull request by ramizbalayil**

### CODE REVIEW

File: layers/ChapterLayer.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* global ChapterLayer: true, cc, ccui, res
+*/
+ChapterLayer = ccui.Widget.extend({
```
Comment (rampr) [inline layers/ChapterLayer.js]: Are we not adding namespaces to these global variables ?

Comment (ramizbalayil) [reply layers/ChapterLayer.js]: We haven't done that yet. Should I start with this PR?

Comment (ramizbalayil) [reply layers/ChapterLayer.js]: Will update it with this PR

**Review by rampr — APPROVED** [review]

File: layers/ChapterLayer.js (hunk 1 above)
Comment (ramizbalayil) [reply layers/ChapterLayer.js]: done

**Review by vivekb91 — APPROVED** [review]

**Review by nasimafeef — APPROVED** [review]

**Review by hatim-developer — APPROVED** [review]

**Pull request merged**